bindings/matrix-sdk-ffi/build.rs

```diff
@@ -1,3 +1,34 @@
+use std::env;
+
+/// Adds a temporary workaround for an issue with the Rust compiler and Android
+/// in x86_64 devices: https://github.com/rust-lang/rust/issues/109717.
+/// The workaround comes from: https://github.com/mozilla/application-services/pull/5442
+fn setup_x86_64_android_workaround() {
+    let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap();
+    let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
```

Comment:
Can you use `.expect()` here with a message similar to the below for `ANDROID_NDK_HOME`?